tox.ini

```diff
@@ -0,0 +1,38 @@
+[tox]
+envlist = commit
+          prospector
+          security
+          changes
+          coverage
+
+[testenv]
+description = Run tests
+basepython = python3.6
+deps = GitPython~=2.1
+passenv = FILES_TO_LINT
+
+[testenv:prospector]
+description = Run the Prospector test
+deps = prospector>=1.1
+commands = /bin/sh -c 'if [ -z $FILES_TO_LINT ]; then echo "No .py files to lint"; else echo $FILES_TO_LINT | xargs prospector; fi'
+           /bin/sh -c 'echo $FILES_TO_LINT'
+
+[testenv:commit]
+description = Run the Commit test
+commands = python ci/test_commit_message.py
+
+[testenv:security]
+description = Run bandit
+deps = bandit~=1.6
+commands = /bin/sh -c 'if [ -z $FILES_TO_LINT ]; then echo "No .py files to lint"; else echo $FILES_TO_LINT | xargs bandit; fi'
+
+[testenv:changes]
+description = Run changes
+commands = python ci/test_files_touched.py
+
+[testenv:coverage]
+description = Get current coverage
+deps = coverage
+       pytest
+commands = coverage run -m pytest tests
+           coverage report --omit .tox/**,tests/**
```

Comment:
No need to add the `pytest` dependency. You can run `coverage run -m unittest discover -s tests`.